build.sh

```diff
@@ -529,13 +529,17 @@ function processSource {
   BUILD_TYPE="$2"
```

Comment:
It might even be nice to prompt people that aren't using `PROCESS_WITH_RUST=1` to do so, so we can get more testing? Or maybe we're not quite ready yet, not sure..

Reply:
It seems like the more likely deployment for most would be to have the whatwg build server support it, and have clients upload the relevant files locally (probably zipped). In that case, most end-users won't need to have a Rust toolchain available.